scripts/docs.js

```diff
@@ -91,8 +91,15 @@
   $('h2,h3').filter('[id]').each(function () {
     // check if existing link
     if (!$(this).children(0) || $(this).children(0).prop('tagName') !== 'A') {
-      $(this).html('<a href="#'+$(this).attr('id')+'">' + $(this).text() + '</a>');
+      $(this).html('<a class="smooth-scroll" href="#'+$(this).attr('id')+'">' + $(this).text() + '</a>');
     }
   });
 
+  $('a.smooth-scroll').on('click', function(event) {
+      event.preventDefault();
+      var $anchor = $(this);
+      $('html, body').stop().animate({
```

Comment:
dono should copied from online - maybe http://stackoverflow.com/questions/19303405/difference-between-html-body-animate-and-body-animate